lmcache/storage_backend/__init__.py

```diff
@@ -40,11 +41,19 @@ def CreateStorageBackend(config: LMCacheEngineConfig,
                         f" backend")
 
                     return LMCLocalBackend(config, mpool_metadata, dst_device)
-                case _:
+                case "global_disk":
```

Comment:
No, it simply means a disk that can be shared among different LM Engines. If you can use a virtual disk like `/dataheart`, it works the same way. The disk doesn’t need to be physically present; as long as you can interact with it like a normal disk, it can be utilized.

Additionally, it complements the LocalDisk in both name and functionality.

For a remote server, you'll serialize it using our own protocol. So, I believe it’s independent of the disk backend.